src-docs/src/views/inline_edit/inline_edit.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+
+import {
+  EuiInlineEdit,
+  EuiSpacer,
+  EuiButtonGroup,
+  EuiTextProps,
+} from '../../../../src';
+
+export default () => {
+  // Display Text Toggles
+  const textSizeButtons = [
+    {
+      id: 'xs',
+      label: 'Extra Small',
+    },
+    {
+      id: 's',
+      label: 'Small',
+    },
+    {
+      id: 'm',
+      label: 'Medium',
+    },
+    {
+      id: 'relative',
+      label: 'Relative',
+    },
+  ];
+
+  const [toggleTextButtonSize, setToggleTextButtonSize] = useState<
+    EuiTextProps['size']
+  >('m');
+
+  const textOnChange = (optionId: EuiTextProps['size']) => {
+    setToggleTextButtonSize(optionId);
+  };
```

Comment:
var/function name suggestions - i think these are both more succinct and syntactically descriptive

```suggestion
  const [textSize, setTextSize] = useState<EuiTextProps['size']>('m');

  const onTextSizeChange = (optionId: EuiTextProps['size']) => {
    setTextSize(optionId);
  };
```

Reply:
Same comment/change request for the title example as well (but change `text` to `title` where applicable)